college_flow_app/lib/features/review/presentation/review_form.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:college_flow_app/config/design_system/data/colors/colors.dart';
+import 'package:college_flow_app/config/design_system/data/object_styles/object_styles.dart';
+import 'package:college_flow_app/features/review/presentation/widgets/textfields/flow_text_field.dart';
+import 'package:college_flow_app/features/review/presentation/widgets/textfields/flow_text_field_variant.dart';
+import 'package:flutter_rating_bar/flutter_rating_bar.dart';
+import 'package:college_flow_app/config/design_system/data/spacing/spacing.dart';
+import 'package:college_flow_app/config/routes/flow_routes.dart';
+import 'package:college_flow_app/features/review/presentation/widgets/header.dart';
+import 'package:college_flow_app/shared/widgets/buttons/flow_button.dart';
+import 'package:college_flow_app/shared/widgets/flow_icon.dart';
+import 'package:college_flow_app/shared/widgets/gap.dart';
+import 'package:flutter/material.dart';
+
+class ReviewForm extends StatefulWidget {
+  const ReviewForm({Key? key}) : super(key: key);
+
+  @override
+  State<ReviewForm> createState() => _ReviewFormState();
+}
+
+class _ReviewFormState extends State<ReviewForm> {
+  final _pageViewController = PageController();
+  final String _mockSubjectName = 'Avaliação';
+  final String _mockSubjectCode = 'Avalie uma Disciplina';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        elevation: 0,
+        backgroundColor: colorPrimary,
+      ),
+      backgroundColor: colorWhite,
+      body: SafeArea(
+        child: Column(
+          children: [
+            Header(
+              subjectCode: _mockSubjectCode,
+              subjectName: _mockSubjectName,
+            ),
```

Comment:
if Header is being used in other contexts rather than the subject change it's params name. I think I could be `title` and `description`